pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -547,6 +547,22 @@ func init() {
 			testName: "bz-kube-apiserver][invariant] alert/KubeAPIErrorBudgetBurn should not be at or above info",
 		}] = "history correction on kube update, expires 2022-01-24"
 	}
+
+	for _, jobName := range []string{
+		"periodic-ci-openshift-multiarch-master-nightly-4.10-upgrade-from-nightly-4.9-ocp-remote-libvirt-s390x",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-aws-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-e2e-azure-upgrade-single-node",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-ovn-upgrade",
+		"periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-upgrade",
+		"periodic-ci-openshift-release-master-nightly-4.10-upgrade-from-stable-4.9-e2e-metal-ipi-upgrade",
```

Comment:
Some of these are not jobs we aggregate on but we didn't exactly make that clear this morning, I didn't even realize. The multiarch one for sure can go. Probably the single-node, and the metal job, none of those should be aggregated, but may want to wait to see what David thinks.

Reply:
Bring up the results in the test comparison view.  You only need to force skip on ones below about 50% pass rate.  And be sure that blocker bugs are open and don't let them be closed on, "no longer failing in CI".

Reply:
Test comparison view: https://datastudio.google.com/reporting/519bfc5e-6996-46bf-9269-0e2ca6d95eb4/page/qKmaC

Filter on test name contains "sandboxes by other".

Here's the blocker for at least aws-own: https://bugzilla.redhat.com/show_bug.cgi?id=2038481

The only one below 50% in that chart is aws ovn as far as I can tell from this.

Reply:
I think it's just periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-aws-ovn-upgrade

We failed a gcp ovn but it was only off by one pass, 8 when we required 9, so that's not the same issue at all.